templates/dashboard/reviews/given-reviews.php

```diff
@@ -87,7 +85,7 @@
 						</div>
 
 						<div class="tutor-fs-6 tutor-color-muted">
-							<?php echo htmlspecialchars( stripslashes( $review->comment_content ) ); ?>
+						<?php echo wp_kses_post( htmlspecialchars( stripslashes( $review->comment_content ) ) ); ?>
```

Comment:
We use textarea for Review content, so instead of wp_kses_post we can use esc_textarea

Reply:
Done